data/events/2023-melbourne.yml

```
@@ -211,6 +210,11 @@ program:
     date: 2023-03-16T00:00:00+10:00
     start_time: "16:45"
     end_time: "17:00"
+  - title: "Evening Event @ Ballers Clubhouse"
+    type: workshop
```

```suggestion
    type: custom
```

Technically not a workshop, also this should fix render errors